Signal/src/ViewControllers/HomeView/HomeViewCell.m

```diff
@@ -184,22 +172,22 @@ - (void)configureWithThread:(TSThread *)thread
     [self updateNameLabel];
     [self updateAvatarView];
 
+    self.payloadView.spacing = 0.f;
+    self.topRowView.spacing = ceil([HomeViewCell scaleValueWithDynamicType:5]);
```

Comment:
Myles wanted slightly different spacing between the items in this row, but I think this'll be fine.  I'll verify with him tomorrow.